admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -71,17 +71,27 @@
       :selected-files="selectedFiles"
       :dropzone="dropzone"
       @unselectAll="unselectAll"
-      @removeSelection="removeSelection"
+      @removeSelection="toggleModal"
       @selectAll="selectAll"
       @saveSelectedFile="saveSelectedFile"
       @replacedFile="manageReplacedFile"
-      @coverChanged="changeCover"
       :files="files"
       :locales="locales"
       :selected-locale="selectedLocale"
       :loading="buttonLoading"
     />
 
+    <modal
+      v-if="showModal"
+      :confirmation="true"
+      :modal-title="$t('modal.title', {
+        '%filesNb%': this.selectedFiles.length,
+      })"
+      :confirm-label="$t('modal.accept')"
+      :cancel-label="$t('modal.close')"
+      @confirm="removeSelection"
```

Comment:
Shouldn't you handle the close as well?

```suggestion
      @confirm="removeSelection"
      @close="showModal = false"
```

Or something?

Reply:
Not needed, if the user click on cancel, it just doesn't run into removeSelection and nothing change